src/Server/OutputBuilder.cs

```diff
@@ -41,21 +66,17 @@ public OutputBuilder(int initialCapacity = 32)
         /// <returns></returns>
         public void ErrorMessage(string error)
         {
-            AppendSeparator('=', "red", true, error.Length);
-            AppendLine($"<%red%>{error}<%n%>");
-            AppendSeparator('=', "red", true, error.Length);
+            AppendLine($"<%error%>{error}");
```

Comment:
This strategy would be broken for errors or warnings which include multiple lines of output.
nit: This also skipped the fluent syntax (returning OutputBuilder) that most the other API uses.